Client/Frontend/Share/ShareExtensionHelper.swift

```diff
@@ -39,8 +45,18 @@ class ShareExtensionHelper: NSObject {
             activityItems.append(TitleActivityItemProvider(title: title))
         }
         activityItems.append(self)
+        
+        if let devices = self.profile.remoteClientsAndTabs.getRemoteDevices().value.successValue?.count {
+            for i in 0 ... devices-1 {
```

Comment:
If you used swift iteration would be better 
`for device in devices'

Reply:
I remember doing it like this at the beginning but then I guess I got an error that strides were not supported when I used 
`for device in self.profile.remoteClientsAndTabs.getRemoteDevices().value.successValue?.count`

and of course, it had to be unwrapped first because it's optional and then you can't iterate on it unless there are devices available. but will check it one more time to make it better. 

Thanks for taking the time to review.

Reply:
![image](https://user-images.githubusercontent.com/11700198/78519396-82805d00-77c3-11ea-9f55-093c03b61a6e.png)
@YoloAbdo